packages/zosfiles/src/methods/upload/Upload.ts

```diff
@@ -242,7 +246,11 @@ export class Upload {
         const uploadRequest: IRestClientResponse = await ZosmfRestClient.putExpectFullResponse(session, requestOptions);
 
         // By default, apiResponse is empty when uploading
-        const apiResponse: any = {};
+        const apiResponse: any = {
+            success: true,
+            from: "Stream<>",
```

Comment:
Nitpicky comment...
Can we surround the word Steam (or buffer) in the angle brackets? 
`<Stream>` or `<Buffer>`